### ISSUE TITLE
claymorecryptonightamd producing lowdifficulty shares on miningpoolhub cryptonightv7

### ISSUE THREAD

**Issue by aaronsta1** [MultiPoolMiner/MultiPoolMiner#2229]
after about an hour of mining i have gotten zero shares and no payout on xmr.. this is the log.

`XMR: 11/08/18-03:15:15 - SHARE FOUND - (GPU 0; share value 0x00019730)
cannot find block data:
{"id":4,"jsonrpc":"2.0","error":{"code":-1,"message":"Lowdifficultyshare"},"result":null}
cannot convert blob data
GPU0 t=57C fan=43%, GPU1 t=40C fan=61%, GPU2 t=49C fan=48%, GPU3 t=41C fan=60%, GPU4 t=39C fan=62%
XMR: 11/08/18-03:15:46 - SHARE FOUND - (GPU 3; share value 0x00011fc7)
cannot find block data:
{"id":4,"jsonrpc":"2.0","error":{"code":-1,"message":"Lowdifficultyshare"},"result":null}
cannot convert blob data
GPU0 t=58C fan=43%, GPU1 t=41C fan=61%, GPU2 t=49C fan=48%, GPU3 t=42C fan=61%, GPU4 t=39C fan=61%
XMR: 11/08/18-03:15:52 - SHARE FOUND - (GPU 3; share value 0x00013957)
cannot find block data:
{"id":4,"jsonrpc":"2.0","error":{"code":-1,"message":"Lowdifficultyshare"},"result":null}
cannot convert blob data
XMR: 11/08/18-03:15:54 - SHARE FOUND - (GPU 2; share value 0x000102fe)
cannot find block data:
{"id":4,"jsonrpc":"2.0","error":{"code":-1,"message":"Lowdifficultyshare"},"result":null}
cannot convert blob data
XMR: 11/08/18-03:16:14 - SHARE FOUND - (GPU 2; share value 0x0001833c)
cannot find block data:
{"id":4,"jsonrpc":"2.0","error":{"code":-1,"message":"Lowdifficultyshare"},"result":null}`

**Comment by aaronsta1** [comment]
i found this
2) Claymore GPU Miner (Cryptonight v8 algo not supported yet! Do not use this miner after 18th Oct)


so i guess you cant use that miner on MPH.

**Comment by UselessGuru** [comment]
@aaronsta1 

Are you using a recent algorithms.txt? 

Just for a test install Beta 9 to an new directory, delete all miner files except AMD-ClaymoreCryptonight_v11.3.ps1, then copy your config file and start MPM.

In my Beta 9 it just mines CryptonightV7.

**Comment by aaronsta1** [comment]
ok so beta 9 is benchmarking monero cryptonightv8.. sweet.. 

you know pheonix miner 3.5d has an autotune feature and to get the benchmark right it needs longer time..

**Comment by aaronsta1** [comment]
getting an error but its still benchmarking.

`Write-Log : Miner (AMD-SgminerAvermore_v1.4.1-GPU#00-GPU#01-GPU#02-GPU#03-GPU#04 {X16R@Zpool}) has failed.
At C:\Users\user\Documents\MultiPoolMiner_3_1_0_Beta9\MultiPoolMiner.ps1:893 char:17
+ ...             Write-Log -Level Error "Miner ($($Miner.Name) {$(($Miner. ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
+ FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Write-Log`

**Comment by UselessGuru** [comment]
> you know pheonix miner 3.5d has an autotune feature and to get the benchmark right it needs longer time

Yes, I do, but it is not really helpful with MPM because there > It will re-optimize again and again. :-(

**Comment by aaronsta1** [comment]
that write log error keeps popping up over and over but its still seems to be working.. ill let it go for a few hours and see what happens.

thanks

**Comment by UselessGuru** [comment]
> getting an error but its still benchmarking.

https://github.com/MultiPoolMiner/MultiPoolMiner/issues/2208#issuecomment-432696257
